commands/project.go

```diff
@@ -0,0 +1,98 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package commands
+
+import (
+	"github.com/apache/incubator-openwhisk-cli/wski18n"
+	"github.com/apache/incubator-openwhisk-wskdeploy/cmd"
+	"github.com/apache/incubator-openwhisk-wskdeploy/utils"
+	wskdeploy_wski18n "github.com/apache/incubator-openwhisk-wskdeploy/wski18n"
+	"github.com/spf13/cobra"
+)
+
+var projectCmd = &cobra.Command{
+	Use:   "project",
+	Short: "The OpenWhisk Project Management Tool",
+}
+
+var projectDeployCmd = &cobra.Command{
+	Use:           "deploy",
+	Short:         wski18n.T(wskdeploy_wski18n.ID_CMD_DESC_SHORT_ROOT),
+	Long:          wski18n.T("help message for deployment"),
+	SilenceUsage:  true,
+	SilenceErrors: true,
+	RunE: func(cobraCMD *cobra.Command, args []string) error {
+		return cmd.Deploy(cobraCMD)
+	},
+}
+
+var projectUnDeployCmd = &cobra.Command{
+	Use:           "undeploy",
+	Short:         wski18n.T(wskdeploy_wski18n.ID_CMD_DESC_SHORT_UNDEPLOY),
+	Long:          wski18n.T("help message for undeployment"),
```

Comment:
I think this should use a token identifier like on the previous line.
There is a file started in the cli to collected all the identifiers.

Reply:
> I think this should use a token identifier like on the previous line.
> There is a file started in the cli to collected all the identifiers.

Didn't find such file, have introduced `i18n_ids.go` file following `wskdeploy`, if it looks good, I will convert all the messages from `en_US` to use IDs.

Reply:
https://github.com/apache/incubator-openwhisk-cli/blob/master/commands/messages.go